src/modules/User.js

```diff
@@ -0,0 +1,26 @@
+import { DataTypes } from 'sequelize';
+import { client } from '../untils/db.js';
+
+export const User = client.define('users', {
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  activationToken: {
+    type: DataTypes.STRING,
```

Comment:
Consider adding `allowNull: true` for `activationToken` to explicitly indicate that this field can be null, which is often the case after activation.